vita3k/modules/SceDriverUser/SceAppMgrUser.cpp

```diff
@@ -344,9 +382,12 @@ EXPORT(int, sceAppMgrGetSystemDataFilePlayReady) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppMgrGetUserDirPath) {
-    TRACY_FUNC(sceAppMgrGetUserDirPath);
-    return UNIMPLEMENTED();
+EXPORT(int, sceAppMgrGetUserDirPath, int partition_id, char *userDirPath, SceSize path_maxlen) {
+    TRACY_FUNC(sceAppMgrGetUserDirPath, partition_id, userDirPath, path_maxlen);
+    STUBBED("Using strncpy");
+    const auto partition = partition_id == 1 ? "ur0:" : "ux0:";
+    strncpy(userDirPath, fmt::format("{}user/{}", partition, emuenv.io.user_id).c_str(), path_maxlen);
```

Comment:
```suggestion
    strncpy(userDirPath, fmt::format("{}user/{}", partition, emuenv.io.user_id).c_str(), path_maxlen);
```
Use fmt::format_to_n

Reply:
done oki